eng/Packages.Data.props

```diff
@@ -168,9 +168,9 @@
     <!-- Other approved packages -->
     <PackageReference Update="Microsoft.Azure.Amqp" Version="2.6.9" />
     <PackageReference Update="Microsoft.Azure.WebPubSub.Common" Version="1.4.0" />
-    <PackageReference Update="Microsoft.Identity.Client" Version="4.69.1" />
-    <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="4.69.1" />
-    <PackageReference Update="Microsoft.Identity.Client.Broker" Version="4.69.1" />
+    <PackageReference Update="Microsoft.Identity.Client" Version="4.70.2" />
+    <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="4.70.2" />
+    <PackageReference Update="Microsoft.Identity.Client.Broker" Version="4.70.2" />
```

Comment:
Let's mention this in the Azure.Identity and Azure.Identity.Broker (just for the broker package) changelogs.